lib/MetaCPAN/Model/Search.pm

```diff
@@ -1,23 +1,23 @@
 package MetaCPAN::Model::Search;
 
-use Moose;
-
-use v5.10;
+use MetaCPAN::Moose;
 
+use Const::Fast qw( const );
 use Log::Contextual qw( :log :dlog );
-use MooseX::StrictConstructor;
 
+use Cpanel::JSON::XS;
 use Hash::Merge qw( merge );
 use List::Util qw( sum uniq );
 use MetaCPAN::Types qw( Object Str );
 use MetaCPAN::Util qw( single_valued_arrayref_to_scalar );
 
+# Read in here to share
```

Comment:
Could we expand the comments here so that it makes sense without having to scan to the `__DATA__` section?